indexer/src/test/java/au/org/aodn/esindexer/service/StacCollectionMapperServiceTests.java

```diff
@@ -349,4 +349,19 @@ public void verifyTitleFreeThemes() throws IOException {
                 "Stac not equals for sample14"
         );
     }
+
+    @Test
+    public void verifyWorks() throws IOException {
```

Comment:
could you make the test method name more meaningful? such as `verifyFindPolygonWorks` so that others know easily what this test case is for.

Reply:
Done please check